src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/IntegerComponent.tsx

```diff
@@ -11,14 +11,35 @@ const IntegerComponent = ({
   onChange,
   label = "Integer",
   structureDefinition,
-  signed = true,
+  unsignedInt = true,
```

Comment:
how does this and all the other refactors related to unsigned vs positive int handle a regular/signed integer (as defined in [https://jira.cms.gov/browse/MAT-7671](https://jira.cms.gov/browse/MAT-7671))

Reply:
@nmorasb Just pushed in the new changes thanks!

Reply:
Integer was done previously in 7671. 7684 was to add support for UnsignedInt, and 7685 was to add support for PositiveInt, but we also still need to keep support for a Signed Integer −2,147,483,648..2,147,483,647 even while adding support for PositiveInt and UnsignedInt, right?

Reply:
@nmorasb Ok I see, I'll add signed type too thanks!

Reply:
@nmorasb I just added Signed Integer support